netket/vqs/exact/expect.py

```diff
@@ -122,9 +124,8 @@ def _exp_forces(
     new_model_state = new_model_state[0] if is_mutable else None
 
     return (
-        None,
+        Stats(mean=expval_O, error_of_mean=0.0, variance=variance),
```

Comment:
why do we want do do this? it is not super intuitive that the exact evolution returns a stat object, in fact it shouldn't I believe :)

Reply:
the ExactState aka `FullSummationState` `expect` method does return a stats object so... I'm simply making the two consistent here.

Reply:
I mainly care about the variance.

Reply:
Anyhow, we set error_of_mean to 0 so it's mathematically correct... I agree with you that it is ugly to return a Stats object but changing this would require more thought than I have time for

Reply:
alright, I see... it's not beautiful but might make things more consistent when you want to compare with MC